src/NzbDrone.Core/Indexers/Definitions/Cardigann/CardigannRequestGenerator.cs

```diff
@@ -1015,8 +1015,14 @@ public bool CheckIfLoginIsNeeded(HttpResponse response)
             return false;
         }
 
-        private IEnumerable<IndexerRequest> GetRequest(Dictionary<string, object> variables)
+        private IEnumerable<IndexerRequest> GetRequest(Dictionary<string, object> variables, int limit, int offset)
```

Comment:
just pass in `searchCriteria` here, then we only need to default the offset and limit once instead of repeating it in every method call above.